Ansible/roles/marvin/tasks/install_marvin_prereqs.yml

```diff
@@ -100,6 +100,19 @@
     - marvin
     - marvin_install
 
+- name: remove yum openSSL package
+  yum:
+    name: pyOpenSSL
+    state: absent
+
+- name: Install pyOpenSSL with pip
+  pip: name=pyOpenSSL extra_args='--upgrade'
+  with_items:
+  - pyopenssl
+  tags:
+    - marvin
+    - maring_install
```

Comment:
@borisstoyanov please fix typo `maring` to `marvin`

Reply:
fixed